src/interpreter/ByteCodeInterpreter.cpp

```diff
@@ -70,6 +70,7 @@ Value ByteCodeInterpreter::interpret(ExecutionState& state, ByteCodeBlock* byteC
     {
         char* codeBuffer = byteCodeBlock->m_code.data();
         programCounter = (size_t)(codeBuffer + programCounter);
+        LexicalEnvironment* oldEnv = state.lexicalEnvironment();
```

Comment:
How about move this variable into other place?
I think this variable not used many place but this code block is too hot..

Reply:
I don't know. It is not in a loop, and this restoration is needed when the catch block is executed. Do you have a suggestion?

Reply:
Oops there is no way to resolve the problem